FewCore/Element.swift

```diff
@@ -116,7 +118,9 @@ public class Element {
 				compareAndSetAlpha(view, alpha)
 			}
 			
-			realizedSelf.element = self
+			if old.frame.size != frame.size {
```

Comment:
What makes you say this shouldn't be needed in theory? When the size of a container changes, it seems like its layout should be invalidated.